src/input.ts

```diff
@@ -141,15 +141,14 @@ function arrow(axis: Axis, dir: Direction): Command {
     if (axis != 'horiz' && !sel.empty) return false;
     const end = atEndOfCell(view, axis, dir, true); // the last parameter is to check also caption
     if (axis == 'horiz') {
-      if (end == null) return false;
```

Comment:
@rphillips-cc do you remember exactly why you put this line in? It seemed to be causing problems for me, so I took it out... it seems all good, but maybe you have some extra context?